package.json

```diff
@@ -1,73 +1,27 @@
 {
-  "version": "0.24.0",
-  "name": "solid-repl",
-  "description": "Quickly discover what the solid compiler will generate from your JSX template",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/solidjs/solid-playground.git"
-  },
-  "files": [
-    "lib"
-  ],
-  "homepage": "https://playground.solidjs.com",
-  "author": "Alexandre Mouton-Brady",
-  "module": "lib/index.jsx",
-  "types": "types/types.d.ts",
-  "scripts": {
-    "dev": "vite",
-    "build": "vite build",
-    "preview": "vite preview",
-    "build:lib": "tsc -p tsconfig.build.json && unocss \"./src/**\" && jiti build.ts",
-    "format": "prettier -w .",
```

Comment:
Might be worth keeping this line (and running it, looks like a few files are formatted with tabs instead of spaces)